### DISCUSSION TITLE
Multiple Autosaves

### DISCUSSION THREAD

**Discussion by SlightlyAmazing** [voxeltycoon/issues#791] [General] [answered]
Would be great if multiple autosaves were implemented.

E.G. 
AutoSave1
AutoSave2
AutoSave3

Where it cycles through each one each time.

**Comment by ElvisCooper0815** [comment, marked as answer]
Isn't this already in the game? When i look at my saves i have 3 different Autosaves from the same playthrough.

Or am i missing something completely here?

**Reply by DanielBarreleiro** [reply]
Yeah, I also have tons of autosaves from the same game.

**Reply by SlightlyAmazing** [reply]
oh i may be dumb, lemme check

**Reply by SlightlyAmazing** [reply]
no you are correct. damn.
